server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -3805,11 +3805,18 @@ public UserVm createAdvancedVirtualMachine(DataCenter zone, ServiceOffering serv
     @ActionEvent(eventType = EventTypes.EVENT_VM_CREATE, eventDescription = "deploying Vm")
     public UserVm finalizeCreateVirtualMachine(long vmId) {
         s_logger.info("Loading UserVm " + vmId + " from DB");
-        UserVm userVm = getUserVm(vmId);
+        UserVmVO userVm = _vmDao.findById(vmId);
         if (userVm == null) {
             s_logger.info("Loaded UserVm " + vmId + " (" + userVm.getUuid() + ") from DB");
         } else {
             s_logger.warn("UserVm " + vmId + " does not exist in DB");
```

Comment:
logs in line 3810 and line 3812 seem wrong .
they have been fixed in 4.20

Reply:
done